src/js/actions/search/commands.js

```diff
@@ -123,13 +123,11 @@ define(function (require, exports) {
     var _menuCommandSearchOptions = function () {
         var menuStore = this.flux.store("menu"),
             menu = menuStore.getApplicationMenu(),
-            menuMap = menu.rootMap,
             roots = menu.roots.reverse();
 
         var menuCommands = [];
         roots.forEach(function (root) {
-            var currRoot = menuMap.get(root.id),
```

Comment:
Looks to me like it was just a misunderstanding during the original implementation.... like maybe it seemed that it was necessary to get the root from the map, event though it wasn't.